### PR TITLE
Prep site scaffold for ManilaJS 016.

### PR DESCRIPTION
**Pull request by richardneililagan**
Essentially just reset event landing page meta, plus new hero image.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -107,7 +109,9 @@ fun night of talks and camaraderie—beginners and experts welcome.
 {% include location-box.html venue=page.venue %}{% else %}
 {% include location-box-tbd.html %}{% endif %}
 
+{% if page.venue %}
```
Comment (richardneililagan) [inline index.md:112]: Also added this --- so the gratitude text doesn't show up when `venue` is not final. :p

Comment (rstacruz) [inline index.md:112]: :+1:

File: index.md [hunk 2]
```diff
@@ -1,62 +1,63 @@
...
   color: "#203a4f"
   opacity: 0.20
-  image: images/015/bg.jpg
-  # attribution_source: ""
-  # attribution_url: ""
+  image: images/016/bg.jpg
+  # :: current image is under Creative Commons Zero however,
+  #    so no attribution is required. Might as well though, yeah?
+  attribution_source: "NASA"
+  attribution_url: "https://unsplash.com/photos/NuE8Nu3otjo"
```
Comment (rstacruz) [inline index.md:28]: btw we can use unsplash.it haha

File: index.md [hunk 3]
```diff
@@ -1,62 +1,63 @@
...
+  date: 2016-02-05
   time: 7PM
-  location: Silicon Valley Insight, BGC
+  location: VentureSpace PH, Shaw Boulevard
 
 registration:
-  date: 2015-08-03
-  state: over # announce | open | closed | over
-  signup: "http://goo.gl/forms/xH2LVnhmDl"
+  # date:
```
Comment (rstacruz) [inline index.md]: `2016-02-01` — monday of that week.

File: index.md (hunk 2 above)
Comment (richardneililagan) [inline index.md:28]: TIL haha! Would you prefer we host the image ourselves, or just link from unsplash.it?

Comment (rstacruz) [inline index.md:28]: kahit dito na. i'll think of another image hehe

**Comment by rstacruz** [comment]
Merged in #17, thanks for your help!

**Comment by richardneililagan** [comment]
No problemo! :+1:

**Pull request merged**